infra/feast-operator/README.md

```diff
@@ -0,0 +1,11 @@
+# Feast Feature Server Helm-based Operator
+
+Leverages [operator-sdk](https://github.com/operator-framework/operator-sdk) and the [feast-feature-server helm chart](/infra/charts/feast-feature-server).
+
+### To run against an K8S cluster -
+```bash
+$ make deploy
+
```

Comment:
Could we add getting started docs for this using Kind?

Reply:
I meant this documentation: https://kind.sigs.k8s.io/docs/user/quick-start/

Probably a lot of users may not be familiar with K8s and so a quick intro with the minimum commands to be able to run it locally is useful.

Reply:
we aren't using the `kind` tool here so i worry those docs would confuse the user.

Reply:
we aren't using the `kind` tool here so i worry those docs would confuse the user

Reply:
maybe you're refering to the [CRD kind](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)?

Reply:
Kind is another tool for running local kubernetes clusters (like minikube). Kind is the recommended for Kubeflow so I figured it'd be a good thing for us to add some docs on it but that's fine.

Reply:
Here's an example: https://github.com/kubeflow/training-operator/blob/master/docs/development/developer_guide.md